man/memkind.3

```diff
@@ -748,30 +748,34 @@ across all high bandwidth nodes and transparent huge pages are
 disabled.
 .TP
 .B MEMKIND_DAX_KMEM
-Allocate from the closest persistent memory NUMA node(s) at time
-of allocation. If there is not enough memory in closest persistent memory NUMA node(s) to satisfy the request
+Allocate from the closest persistent memory NUMA node at time
+of allocation. If there is not enough memory in closest avaliable persistent memory NUMA node to satisfy the request
 .I errno
 is set to
 .B ENOMEM
 and the allocated pointer is set to
 .IR "NULL" .
 .TP
 .B MEMKIND_DAX_KMEM_ALL
-Allocate from the closest persistent memory NUMA node available at the time
-of allocation. If there is not enough memory on any of persistent memory NUMA nodes to satisfy the request
+Same as
+.B MEMKIND_DAX_KMEM
+except that if there is not enough memory in the closest persistent memory NUMA node to satisfy
+ the request, tha allocation will fall back on next closest persistent memory numa node(s), if avaliable.
```

Comment:
fall back **to the** next...
available

Reply:
I fixed "the available mistake", but please take a look at the explanation of the "fall back on" phrase
in the dictionary: [macmillandictionary.com](https://www.macmillandictionary.com/dictionary/british/fall-back-on).